hbase-server/src/main/java/org/apache/hadoop/hbase/ipc/ServerRpcConnection.java

```diff
@@ -87,11 +87,11 @@
 @edu.umd.cs.findbugs.annotations.SuppressWarnings(value = "VO_VOLATILE_INCREMENT",
     justification = "False positive according to http://sourceforge.net/p/findbugs/bugs/1032/")
 @InterfaceAudience.Private
-abstract class ServerRpcConnection implements Closeable {
+abstract class ServerRpcConnection<T extends RpcServer> implements Closeable {
```

Comment:
This is for letting Netty related classes can get NettyRpcServer without casting?

Reply:
Correct. I can just do the casting if you want, just felt a little nicer. Happy to change if desired